### DISCUSSION TITLE
Skip validation for one field, not all the schema

### DISCUSSION THREAD

**Discussion by samislam** [Automattic/mongoose#12005] [Q&A] [answered]
I have the following long schema:

```js

    const mySchema = new mongoose.Schema({
      // some stuff, email, lastName ... etc
      firstName: {
        type: String,
        required: true,
      }
      password: {
        type: String,
        minLength: 8,
        maxLength: 120,
      }
    })

```

And I am inside one of the routes, I want to save a document that I am editing, but I only want to skip one validation, not the entire schema, I want to skip the validation for the `firstName` field, but I want the rest of the fields to be validated.

Is there something that I can do in Mongoose such as:

```js
    userDoc.firstName = "new first name";
    const newDoc = await userDoc.save({ validateBeforeSave: yes, but not for firstName })
```

Is there something like that?

**Comment by vkarpov15** [comment, marked as answer]
Yeah, check out the pathsToSkip option for validate: https://mongoosejs.com/docs/api/document.html#document_Document-validate
